Tenduke.Client.WPF/Tenduke.Client.WPF.csproj

```diff
@@ -39,16 +39,16 @@
     <None Include="..\10duke-192x192.png" Pack="true" Visible="false" PackagePath="\" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
-    <PackageReference Include="CefSharp.Common" Version="129.0.110" />
-    <PackageReference Include="CefSharp.Wpf" Version="129.0.110" />
+    <PackageReference Include="CefSharp.Common" Version="135.0.170" />
+    <PackageReference Include="CefSharp.Wpf" Version="135.0.170" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'net8.0-windows'">
-    <PackageReference Include="CefSharp.Common.NetCore" Version="129.0.110" />
-    <PackageReference Include="CefSharp.Wpf.NetCore" Version="129.0.110" />
+    <PackageReference Include="CefSharp.Common.NetCore" Version="135.0.170" />
+    <PackageReference Include="CefSharp.Wpf.NetCore" Version="135.0.170" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="chromiumembeddedframework.runtime.win-x64" Version="129.0.11" />
-    <PackageReference Include="chromiumembeddedframework.runtime.win-x86" Version="129.0.11" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x64" Version="135.0.17" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x86" Version="135.0.17" />

```

Comment:
Should the ARM versioon of the runtime be included here?
It has made its way into the sample but not the library. Should ARM be supported by the library as well as x86 and x64?